src/cljs/token/handlers.cljs

```diff
@@ -28,19 +28,42 @@
     [{:keys [eth] :as db} _]
     (let [rpc-url (:rpc-url eth)
           web3    (connect rpc-url)]
-      (accounts web3
-                (fn [error result]
-                  (when error
-                    #_(println (str "error: " error)))
-                  (re-frame/dispatch
-                    [:set-accounts result])))
       (assoc-in db [:eth :web3] web3))))
 
 (register-handler
+  :start-auto-refreshing
+  (fn [db [_ refresh-timer]]
+    (js/setInterval
+      #(re-frame/dispatch [:refresh-accounts]) refresh-timer)
+    db))
+
+(register-handler
+  :refresh-accounts
+  (fn [{:keys [eth] :as db} _]
+    (let [web3 (:web3 eth)]
+      (when web3
+        (accounts web3
+                  (fn [error result]
+                    (when error
+                      (println (str "error: " error)))
```

Comment:
Better to comment `println` because it can cause an error in prod build. Actually, we could start using `toaenso.timbre` for logging, but it is not required in this PR.